Cargo.toml

```diff
@@ -77,8 +77,13 @@ turbopack-wasi = [
 [workspace.lints.clippy]
 too_many_arguments = "allow"
 
-[profile.dev.package.turbo-tasks-macros]
-opt-level = 3
+# This crate is particularly sensitive to compiler optimizations
+[profile.dev.package.turbo-tasks-memory]
+opt-level = 1
+
+# Set the options for dependencies (not crates in the workspace), this mostly impacts cold builds
+[profile.dev.package."*"]
+opt-level = 1
 
 # Set the settings for build scripts and proc-macros.
 [profile.dev.build-override]
```

Comment:
One problem is that due to precedence ordering issues in cargo, dependencies like `syn` will now be compiled with `opt-level = 1` instead of `opt-level = 3`. I need to perform further benchmarking to determine if this is significant and worth working around.

https://github.com/rust-lang/cargo/issues/9351

Reply:
I guess this will mostly affect macro eval? turbo tasks has a few thousand alone but I would hope that the aggregate wins from not optimizing deps would be greater than the perf loss.

also TY for adding comments. our build configs are under documented